src/GerdaFitter.cc

```diff
@@ -357,7 +367,7 @@ GerdaFitter::GerdaFitter(json outconfig) : config(outconfig) {
                     // it's a user defined file
                     if (it.contains("root-file")) {
                         auto obj_name = iso.value().value("hist-name", elh.key());
-                        auto thh = this->GetFitComponent(it["root-file"].get<std::string>(), obj_name, _current_ds.data);
+                        auto thh = this->GetFitComponent(it["root-file"].get<std::string>(), obj_name, _current_ds.data, rebin_x, rebin_y);
```

Comment:
Why not just rebin pdfs and data in the end like it was before?

Reply:
Because TH1 doesn't have a method Rebiny and I cannot dynamically cast TH1 to TH2. If you have a better option modify to your liking.

Reply:
btw I can cast TH1 to TH2, I do it in plotspectra